databases/week2/Librarydb.sql

```diff
@@ -0,0 +1,55 @@
+-- Create a Library database.
+
+CREATE DATABASE IF NOT EXISTS `Library`;
+  
+  CREATE TABLE    
+ Library.author (
```

Comment:
Hi Haile as for table names isn't better for Space to be replaced with Underscore —

Reply:
where? there is no space it is the database name Library and table name author( Library.author)

Reply:
where? there is no space it is the database name Library and table name author( Library.author)

Reply:
where? there is no space it is the database name Library and table name author( Library.author)